QUARKUS-6258.md

```diff
@@ -40,12 +47,16 @@ According to Mario, this is intentional, since they consider upstream lang4chain
 Ollama modules can be started on the laptop in dev mode. Jlama ones can be run as well.
 The features we want to cover are being used in integration-tests modules for OpenAI (Chats, Chat Memory, user/system messages, Function calling/Tools, Guardrails), RAG,. There is no coverage for WebSockets (only in codestarts) and for ChatModel (only in mocktests).
 
-### Impact on test suite
+## Impact on test suite
 We will focus on updating and running upstream tests from the Quarkiverse extensions against productised binaries.
```

Comment:
Again, this is really vague.

The test plan misses
* what the scenarios do
* what are the pass criteria
* how will the upstream test suite be ran with RHBQ platform
* how long the jobs will run and what infra they require

For full support, this test plan also needs to account for the fact that LTS lives for a long time and the upstream needs some sort of way to check that the backports work properly without introducing regressions, and that the development branch of the extensions is still usable with Quarkus platform in development